x-pack/plugins/features/server/routes/index.ts

```diff
@@ -31,13 +31,19 @@ export function defineRoutes({ router, featureRegistry, getLegacyAPI }: RouteDef
       const allFeatures = featureRegistry.getAll();
 
       return response.ok({
-        body: allFeatures.filter(
-          feature =>
-            request.query.ignoreValidLicenses ||
-            !feature.validLicenses ||
-            !feature.validLicenses.length ||
-            getLegacyAPI().xpackInfo.license.isOneOf(feature.validLicenses)
-        ),
+        body: allFeatures
+          .filter(
+            feature =>
+              request.query.ignoreValidLicenses ||
+              !feature.validLicenses ||
+              !feature.validLicenses.length ||
+              getLegacyAPI().xpackInfo.license.isOneOf(feature.validLicenses)
```

Comment:
(outside of the scope of the PR) @restrry: can this be replaced with a call to one of the `licensing` plugin API now?

Reply:
I believe so, `licensing plugin exposes `license$`